cmake/CMakeResourceDependencies.cpp

```diff
@@ -20,14 +20,12 @@
 
 =============================================================================*/
 
-#include <iostream>
-
 namespace {
 
 // This is dummy code to silence some linkers warning about
 // empty object files.
 struct CMakeResourceDependencies
 {
-  CMakeResourceDependencies() { std::cout << std::flush; }
+  CMakeResourceDependencies() { }
```

Comment:
The `std::cout` code was added to silence `has no symbols` warnings issued by `ranlib` on MacOS, see commit
bc72d205201930ab9baef8bc203395d548e600f7. Looking at the CI build output, those warnings now appear again.

`ranlib` seems to have a command line option now to suppress those warnings. Maybe we can make use of them?

Reply:
Added an issue to deal with the warning.